tools/workspace-plugin/src/executors/build/executor.spec.ts

```diff
@@ -270,13 +270,19 @@ describe('Build Executor', () => {
 
     const publicApiFilePath = join(workspaceRoot, 'libs/proj/src/index.ts');
     const originalApiContent = readFileSync(publicApiFilePath);
+    const existingEnvVariableCi = process.env.CI;
+
     appendFileSync(publicApiFilePath, `export const hello='new public api';\n`);
+    // force api-extractor CI behaviors to fail on both Local/CI
+    process.env.CI = 'true';
 
-    // force api-extractor to generate api.md and not fail on Local/CI
-    process.env.__FORCE_API_MD_UPDATE__ = '';
     const outputFailed = await executor(options, context);
     expect(outputFailed.success).toBe(false);
 
+    // cleanup
     writeFileSync(publicApiFilePath, originalApiContent, 'utf-8');
+    if (existingEnvVariableCi) {
+      process.env.CI = existingEnvVariableCi;
```

Comment:
won't be better to cleanup, even if the test fails?

```suggestion
    // force api-extractor CI behaviors to fail on both Local/CI
    try {
      const outputFailed = await executor(options, context);
      expect(outputFailed.success).toBe(false);
    } finally {
      // cleanup
      writeFileSync(publicApiFilePath, originalApiContent, 'utf-8');
      if (existingEnvVariableCi) {
        process.env.CI = existingEnvVariableCi;
      }
    }
```

Reply:
i dont think this is gonna work tbh - if assertion is not met jest wont throw an catchable exception via imperative handler. if it does it adds another layer of complexity to the test.